packages/git/src/browser/style/blame.css

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
```

Comment:
Could it be removed?

Reply:
correct, it's a leftover from before adding Styles.

Reply:
done.